dev-python/flexmock/flexmock-0.10.2.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{4,5,6} )
+
+inherit distutils-r1
+
+DESCRIPTION="Testing library to create mocks, stubs and fakes"
+HOMEPAGE="http://flexmock.readthedocs.org/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
+
+DEPEND="
+	"
+RDEPEND="${DEPEND}"
+BDEPEND="${DEPEND}
```

Comment:
`BDEPEND` is hardly ever used across the Portage tree. Please nix it in favour of `DEPEND`.

Reply:
`BDEPEND` is future EAPI 7 variable name, please don't use it.